src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/InbreedingCoeff.java

```diff
@@ -61,6 +63,13 @@ public InbreedingCoeff(final File pedigreeFile){
         super(pedigreeFile);
     }
 
+    @Override
+    void validateArguments(Collection<String> founderIds, File pedigreeFile) {
+        if ((founderIds == null || founderIds.isEmpty()) && pedigreeFile == null) {
+            warning.warn("InbreedingCoeff annotation will not be calculated, no 'founder-id' or 'pedigree' arguments provided");
+        }
+    }
```

Comment:
I'm also unclear on why these warn rather than just throw? The user asked for the annotation, and didn't provide enough input. If we do throw, the descriptor test should have a test added for that.

Reply:
The primary reason for these all being warnings is because there are relatively easy ways in the gatk (like variant annotator i believe) to specify "all annotations" as a holdover from gatk 3. I would think that using that argument shouldn't suddenly require a ped file be provided, rather I would rather be clear about warning the user upfront that these annotations won't be calculated as is the case for a bunch of our other annotations when they encounter unannotatable data (maybe they don't have the reads on hand for example...)

Reply:
I would think it more consistent with other annotations to simply warn the user if it is missing required input.